infrastructure/env/development/tfvars.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
perhaps e should keep a branch with our own config?

Reply:
sorry ? :)

Reply:
this config had some unset vars (values set to `""`), my question is that we should fill them with the actual values of our config (?)

Reply:
our own confg is in the production directory

Reply:
which unset vars ?